File: EjemploCodigo/PHP/SubClasesCar/UberPool.php
<?php
require_once('../Car.php');
require_once('Driver.php');

class UberPool extends Car{
    public $brand;
    public $model;

    public function __construct($license, $driver, $brand, $model ) {
        parent ::__construct($license, $driver); //llamo al metodo consturctor de Car ->  esto es por que Car tiene un constructor 
        $this->brand = $brand;
        $this->model = $model;
    }

}

File: EjemploCodigo/Java/SubClasesCar/UberPool.java
package Java.SubClasesCar;

import Java.Account;
import Java.Car;

public class UberPool  extends Car {
    String brand;
    String model;
    
    public UberPool(String license, Account driver, String brand,  String model){
        super(license, driver);//llamo al metodo consturctor de Car ->  esto es por que Car tiene un constructor 
        this.brand = brand;
        this.model  = model;
    }
}
